doc/setup.md

```diff
@@ -21,21 +21,28 @@ $ elan default leanprover/lean4:nightly
 $ elan override set leanprover/lean4:stable
 ```
 
-### `leanpkg`
+NOTE: there are no `lean4:stable` releases yet, so for now
+just use `leanprover/lean4:nightly`
```

Comment:
There sure are, does it not work for you?

Reply:
As phrased I suppose this is incorrect given that stable releases do exist, but I would support noting that nightly is the recommended version to get. People come to Zulip every now and then with weird issues the solution to which is just "you're on stable and the version is very old, install nightly instead".

Reply:
Well, this PR is now linked to an issue on the next milestone, so let's work on improving that part :) . However, we do actually recommend nightlies in the quickstart.

Reply:
I went with the simpler vscode setup.  Please see new version, thanks.